examples.md

```diff
@@ -321,5 +321,70 @@ Does an equivalent of a `copy run start` **AND** saves a local copy to the local
     password={{ password }}
 ```
 
+# Filter Plugins
+
+##### ntc_parse
+
+Get structured data back using TextFSM along with `ios_command` and `ntc_parse` filter plugin within Ansible.
+```
```

Comment:
Should include the block type for all blocks (including current ones that don't have it)
```
|```yaml
content
|```
```